debezium-microbenchmark-oracle/src/main/java/io/debezium/performance/connector/oracle/EndToEndPerf.java

```diff
@@ -252,7 +252,7 @@ private JdbcConfiguration testJdbcConfig() {
 
         private OracleConnection getTestConnection() {
             Configuration config = testConfig().build();
-            Configuration jdbcConfig = config.subset("database.", true);
+            JdbcConfiguration jdbcConfig = JdbcConfiguration.adapt(config.subset("database.", true));
```

Comment:
Could we call `testJdbcConfig()` instead?

Reply:
@gunnarmorling Yes, you are right, code updated.